react/features/filmstrip/components/Filmstrip.web.js

```diff
@@ -34,11 +34,7 @@ export default class Filmstrip extends Component {
                                 id = 'localAudio'
                                 muted = { true } />
                             <div className = 'videocontainer__toolbar' />
-                            <div className = 'videocontainer__toptoolbar'>
-                                <span
-                                    className
-                                        = 'connection-indicator-container' />
```

Comment:
connection-indicator-container was used temporarily to hold the ConnectionIndicator React component while other, non-React indicators were appended to videocontainer__toptoolbar. With the indicators now in React, videocontainer__toptoolbar can be container for all of them.